driver-core/src/main/com/mongodb/client/model/Filters.java

```diff
@@ -73,7 +96,19 @@ private Filters() {
      * @mongodb.driver.manual reference/operator/query/ne $ne
      */
     public static <TItem> Bson ne(final String fieldName, final TItem value) {
-        return new OperatorFilter<TItem>("$ne", fieldName, value);
+        return new SimpleEncodingFilter<Bson>(fieldName, ne(value));
+    }
+
+    /**
+     * Creates a bson document for the $gt operator
+     *
+     * @param value the value
+     * @param <TItem> the value type
+     * @return the filter
+     * @mongodb.driver.manual reference/operator/query/gt $gt
+     */
+    public static <TItem> Bson gt(final TItem value) {
```

Comment:
So now this works: 

    elemMatch("x", and(gt(80), lte(90)))

whereas before you had to resort to 

    elemMatch("x", and(new Document("$gt, 80), new Document("$lt", 90)))

What might be surprising to a user is that you still have to do:

   and(gt("x", 80), lt("x", 90))) 

in order to get

   {x, {$gt : 80, $lt : 90} }

We could add: 

    and(String fieldName, Bson... values)

so a user could do:

    and("x", gt(80), lte(90))

But with both not and and it's easy to get runtime errors, because not all filters are composable:

 not('x', eq('x', 1)))   // not right
 not('x', eq('y', 1)))   // even worse